Fast-follow for latest version (#5033)
fishtown-analytics/docs.getdbt.com: website/docs/reference/commands/version.md

id: "version"
---

The `--version` command-line flag returns information about the currently installed version of dbt Core or the dbt Cloud CLI. This flag is not supported when invoking dbt in other dbt Cloud runtimes (for example, the IDE or scheduled runs).

> - **dbt Core** &mdash; Returns the installed version of dbt-core and the adapters (which is one local bundle).

Reviewer: I think with v 1.8 these are installed separately now and we've decoupled core install from adapters. See [this](https://docs-getdbt-hmoknlxj2-dbt-labs.vercel.app/docs/core/pip-install#installing-the-adapter) for more info and make sure to select v 1.8!
```suggestion
- **dbt Core** &mdash; Returns the installed version of dbt-core and the adapters (which is one local bundle).
```

Reviewer 2: @runleonarun Good callout! Maybe something like:
```suggestion
- **dbt Core** &mdash; Returns the installed version of dbt-core and all adapters. Starting with v1.8, dbt-core and adapter minor versions may diverge.
```
That's also worth updating here:
- https://docs.getdbt.com/docs/dbt-versions/core#how-we-version-adapter-plugins

Author: thanks y'all!! 

since the following section **Version** links to "How dbt Core uses semantic versioning" which has the subsection "How we version adapter plugins" (which already talks about independent releases and minor version), i updated that bullet item mainly to remove "one bundle". as a best practice in user docs, we want to manage link-outs so it's not repetitive on such a short page

Reviewer 2: > "How we version adapter plugins" (which already talks about independent releases and minor version)

Sorry - this was me remembering that we will need to update this page in advance of v1.8 final release. I just added a comment to this issue (I think related), but can open a new issue to track this follow-up work:
- https://github.com/dbt-labs/docs.getdbt.com/issues/5104#issuecomment-2035100934

- **dbt Cloud CLI** &mdash; Returns the installed version of the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) and, for the other `dbt_version` values, the _latest_ version of the dbt runtime in dbt Cloud.

## Versioning
To learn more about release versioning for dbt Core, refer to [How dbt Core uses semantic versioning](docs/dbt-versions/core#how-dbt-core-uses-semantic-versioning). 

> If the [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) setting is selected in dbt Cloud, then `dbt_version` will be the latest (continuous) release version. This also follows semantic versioning guidelines, in the format `YYYY.xx.yy`, with the major version representing the year (for example, `2024.04.1234`).

Reviewer 3: sugg active voice
```suggestion
If dbt Cloud is set to [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version), then `dbt_version` uses the latest (continuous) release version. This also follows semantic versioning guidelines, using the `YYYY.xx.yy` format, where the year is the major version (for example, `2024.04.1234`)
```

Reviewer 3: i'm not sure if i understand where the `1234` comes from or what it means - if its semantic versioning, is it the patch? i thought the semantic version guidelines were more like 'MAJOR.MINOR.PATCH (e.g., 1.0.0)' format? idk I'm not a semantic version expert but the 1234 may confuse users?

Author: @mirnawong1 : "If dbt Cloud is set to..." is still passive voice (not active voice). i really like your sugg for conciseness, thanks!
